### ISSUE TITLE
Provide a "cheap" option for beamforming MWAX -> VDIF by doing FFT instead of PFB

### ISSUE THREAD

**Issue by robotopia** [CIRA-Pulsars-and-Transients-Group/vcsbeam#12]
Currently, if the input is MWAX coarse channelised data, and the desired output is VDIF, then the data are first channelised to 10 kHz via a PFB, beamformed, and then converted back to coarse channels vi the synthesis filter. This might be undesirable because

 1. The currently only available analysis/synthesis filter pairs are not exact inverses of each other, resulting in a small drop in S/N (see [McSweeney et al., 2020](https://ui.adsabs.harvard.edu/abs/2020PASA...37...34M/abstract)), and
 2. It is relatively computationally expensive.

A better option is arguably to use a direct FFT.

**Comment by bwmeyers** [comment]
I suppose this is what you were referring to in our meeting the other day, where we could instead use `cuFFT` and a multiplication by the filter? More memory hungry (~12x mem. footprint for the inversion step) but "simpler".

**Comment by robotopia** [comment]
No, not quite the same thing, but related, I suppose. The idea of this was to do something similar to what Ian (told me he) does in the online beamformer, namely, when starting from coarse channel MWAX data, and wanting to get coarse channel data out, do a cheaper FFT-IFFT pair instead of a PFB-IPFB pair.
